ERCS/erc-7204.md

```diff
@@ -0,0 +1,210 @@
+     */
+    event SimpletokenApprovalForAll(
+        address indexed owner, 
+        address indexed spender,
+        bool approved
+    );
+
+    /**
+     * @notice Approve simple token
+     * @dev Allows spender address to withdraw from your account multiple times, up to the value amount.
+     * @dev If this function is called again it overwrites the current allowance with value.
+     * @dev Emits an {SimpletokenApproval} event.
+     * @param asset Address of the simple token
+     * @param spender Address of the spender
+     * @param value The amount allowed to spend
+     * @return success The bool value returns whether the approve is successful
+     */
+    function simpletokenApprove(address asset, address spender, uint256 value) 
+        external 
+        returns (bool success);
+
+    /**
+     * @notice read simple token allowance value
+     * @param asset Address of the simple token
+     * @param spender Address of the spender
+     * @return remaining The asset amount which spender is still allowed to withdraw from owner.
+     */
+    function simpletokenAllowance(address asset, address spender) 
+        external
+        view
+        returns (uint256 remaining);
+
+    /**
+     * @notice Approve all simple token
+     * @dev Allows spender address to withdraw from your wallet all simple token.
+     * @dev Emits an {SimpletokenApprovalForAll} event.
+     * @param spender Address of the spender
+     * @param approved Approved all tokens
+     * @return success The bool value returns whether the approve is successful
+     */
+    function simpletokenApproveForAll(address spender, bool approved) 
+        external 
+        returns (bool success);
+
+    /**
+     * @notice read spender approved value
+     * @param spender Address of the spender
+     * @return approved Whether to approved spender all tokens
+     */
+    function simpletokenIsApproveForAll(address spender) 
+        external
+        view
+        returns (bool approved);
+
+    /**
+     * @notice Transfer simple token
+     * @dev must call asset.transfer() inside the function
+     * @dev If the caller is not wallet self, must verify the allowance and update the allowance value
+     * @param asset Address of the simple token
+     * @param to Address of the receive
+     * @param value The transaction amount
+     * @return success The bool value returns whether the transfer is successful
+     */
+    function simpletokenTransfer(address asset, address to, uint256 value) 
+        external 
+        returns (bool success); 
+
+    
+
+
+}
+```
+
+
+## Rationale
+
+Use the sequence diagram to compare the difference between using this interface to transfer tokens.
+
+**Alice calls the transfer interface by herself**
+
+The user does not use this ERC to call the transaction sequence diagram(transfer).
+
+```mermaid
+sequenceDiagram
+    autonumber
+    Alice's EOA->>Token Asset Contract: call transfer,update balance state
+ 
+```
```

Comment:
I doubt these will render correctly. If they do not, please replace them with SVG images if possible.